### DISCUSSION TITLE
Update Popper with minor fixes

### DISCUSSION THREAD

**Discussion by ptu14** [floating-ui/floating-ui#1839] [General]
Hi @FezVrasta @atomiks and others,

Is there any chance of releasing a new version of popper e.g. 2.12 with minor fixes?

I most care about:
Strategy "fixed" + pinch-to-zoom generates incorrect popper position in Safari:
https://github.com/floating-ui/floating-ui/pull/1576

I have my library where I have the main version bumped up to `@floating-ui` but the previous main version of my library still needs to work with `@popperjs/core` (compatibility) and the bug in mobile safari is very troublesome.

If you are interested, I am able to do a PR

Best regards

**Comment by atomiks** [comment]
A PR is welcome, thanks

**Reply by ptu14** [reply]
https://github.com/floating-ui/floating-ui/pull/1843
